tests/e2e/ctl_v3_auth_test.go

```diff
@@ -303,29 +296,6 @@ func authTestWatch(cx ctlCtx) {
 
 }
 
-func authTestDefrag(cx ctlCtx) {
```

Comment:
It has been replaced by https://github.com/etcd-io/etcd/blob/main/tests/common/maintenance_auth_test.go#L33-L53